static/js/components/dashboard/FinalExamCard_test.js

```diff
@@ -138,4 +152,34 @@ pay for the course and pass the online work.`;
       'ERROR ERROR'
     );
   });
+
+  it('renders confirm pearson TOS dialog', () => {
+    props.ui.dialogVisibility = { 'pearsonTosDialogVisible': true };
+    renderCard(props);
```

Comment:
before your call to `renderCard` here you need to make sure that `props.program.can_schedule_exam` is `true` - if you do that the dialog will be rendered.